src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -53,6 +58,16 @@
 __all__ = ["TextGenerationPipeline"]
 
 
+class GenerationDefaults:
```

Comment:
nit: would be good to have a small pydoc that explains how the defaults are used/prioritized

Reply:
let's do that in a follow-up